src/assets/static/pgp-worker.js

```diff
@@ -3,12 +3,12 @@ self.window = {crypto: self.crypto}; // to make UMD bundles work
 importScripts('/openpgp.min.js');
 var openpgp = window.openpgp;
 
-var decryptedPrivKeyObj;
+var decryptedPrivKeys = {};
```

Comment:
@Waseem826 we need to create versions of this file as well, let's discuss about it.